src/modules/addNewChain/AddNewChain.tsx

```diff
@@ -0,0 +1,122 @@
+// React and other libraries
+import { FC, useState } from 'react';
+
+import { useSelector } from 'react-redux';
+import { css } from 'styled-components';
+
+import { addNewChainSteps } from './AddNewChain.constants';
+
+// Components
+import { NewAddress } from './components/NewAddress';
+import { ChangeNetwork } from './components/ChangeNetwork';
+import { VerifyAliasChain } from './components/VerifyAliasChain';
+import { Box, Text } from 'blocks';
+import { Stepper } from 'common';
+import UnlockProfileWrapper, { UNLOCK_PROFILE_TYPE } from 'components/chat/unlockProfile/UnlockProfileWrapper';
+
+import { UserStoreType } from 'types';
+
+export type AddNewChainProps = {};
+
+const completedSteps = [0];
+const AddNewChain: FC<AddNewChainProps> = () => {
+  const [activeStepIndex, setActiveStepIndex] = useState<number>(0);
+  const [alias, setAlias] = useState<string>('');
+
+  const { userPushSDKInstance } = useSelector((state: UserStoreType) => state.user);
+
+  const handleNextStep = () => {
+    if (activeStepIndex < 2) {
+      completedSteps.push(activeStepIndex + 1);
+      setActiveStepIndex(activeStepIndex + 1);
+    }
+  };
+
+  return (
+    <Box
```

Comment:
The step completed status should reset if we change the route or after successful creation of chain alias.